scoring/score.py

```diff
@@ -2,15 +2,66 @@ class InvalidScoresheetException(Exception):
     pass
 
 
+CAN_STATE_SCORES = {
+    "B": 3,  # 3 points for can with tape on the Bottom
+    "T": 1,  # 1 point for can with tape on the Top
+    "S": 0,  # 0 points for can on its Side
+}
+
+MAX_CANS = 28
+
+
 class Scorer(object):
     def __init__(self, teams_data, arena_data):
         self._teams_data = teams_data
+        self._arena_data = arena_data
 
     def calculate_scores(self):
-        raise NotImplementedError
+        scores = {}
+        for tla, info in self._teams_data.items():
+            zone = info['zone']
+            cans = self._arena_data[zone]['tokens']
+            points = sum(CAN_STATE_SCORES[c] for c in cans if c != " ")
```

Comment:
Would prefer a descriptive name over 'c', and a more thorough test (perhaps `if c.strip()`?)

Reply:
Would this be preferable?
```suggestion
            points = sum(CAN_STATE_SCORES[y] for x in cans if (y := x.strip()))
```

Reply:
https://github.com/srobo/sr2022-comp/pull/2/commits/273f8351fa4c9ed66344d206f878a7637b39ea7e